templates/prescription/general_list.html

```diff
@@ -131,7 +131,7 @@
 
 <div id="prescription_list">
 
-<form name="presc">
+<form name="presc" method="post" action="{$WEBROOT}/add_rxs_to_list.php">
```

Comment:
hi @scorpions786 ,
Thanks for the contribution.
Is the above script add_rxs_to_list.php missing?
-brady

Reply:
yeah, I missed that file -
just added